charts/pulsar/templates/bookkeeper-cluster-initialize.yaml

```diff
@@ -69,6 +69,9 @@ spec:
                 {{- end }}
                 bin/bookkeeper shell initnewcluster;
             fi
+            {{- if .Values.istio.enabled }}
```

Comment:
I would suggest a different approach:

Can you just introduce a setting called `extraInitCommand`? So you can include this in the init container. Then you can add this command to your values file. This provides better extensibility to the helm chart.